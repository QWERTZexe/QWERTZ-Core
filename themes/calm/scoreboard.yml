## QWERTZ Core Scoreboard Configuration
## https://modrinth.com/plugin/qwertz-core

## 0 is the bottom line, higher numbers are higher on the scoreboard.
## Set a line to "" to effectively remove it.  Use color codes (&)

# Valid placeholders: %server%, %event%, %alive%, %dead%, %countdown%, %player%, %rank%, %wins%, %tokens%
# Also all placeholders in "messages.yml"

# TO MAKE USE OF THIS FILE YOU MUST SET active-theme: "file" IN messages.yml!

title: "\u00A7b\u00A7l%server%" # Scoreboard Title

# The number color to use for the scoreboard numbers. To remove numbers set to "#"
# Hex colors only! e.g. "#ff00ff"
number-color: "#"

lines:
  12: "\u00A7a"  # Strikethrough line
  11: "%colorPrimary%\u00A7l%event%"
  10: "\u00A77\u251C \u00A7f\u1D18\u029F\u1D00\u028F\u1D07\u0280\uA731 \u1D00\u029F\u026A\u1D20\u1D07: %colorPrimary%%alive%"
  9:  "\u00A77\u251C \u00A7f\u1D18\u029F\u1D00\u028F\u1D07\u0280\uA731 \u1D05\u1D07\u1D00\u1D05: %colorPrimary%%dead%"
  8:  "\u00A77\u2514 \u00A7fs\u1D1B\u1D00\u0280\u1D1B\u026A\u0274\u0262 \u026A\u0274 %colorPrimary%%countdown%"
  7:  "\u00A76"
  6:  "%colorPrimary%\u00A7l%player%" # Empty line
  5:  "\u00A7d"
  4:  "%colorPrimary%&l%player%"
  3:  "\u00A77\u2514 \u00A7f\u0280\u1D07\u1D20\u026A\u1D20\u1D07\uA731: %colorPrimary%%tokens%"
  2:  " "
  1:  " " # Strikethrough line
  0:  "" # Bottom line

