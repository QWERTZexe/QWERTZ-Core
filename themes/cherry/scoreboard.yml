## QWERTZ Core Scoreboard Configuration
## https://modrinth.com/plugin/qwertz-core

## 0 is the bottom line, higher numbers are higher on the scoreboard.
## Set a line to "" to effectively remove it.  Use color codes (&)

# Valid placeholders: %server%, %event%, %alive%, %dead%, %countdown%, %player%, %rank%, %wins%, %tokens%
# Also all placeholders in "messages.yml"

# TO MAKE USE OF THIS FILE YOU MUST SET active-theme: "file" IN messages.yml!

title: "&d%server%" # Scoreboard Title

# The number color to use for the scoreboard numbers. To remove numbers set to "#"
# Hex colors only! e.g. "#ff00ff"
number-color: "#"

lines:
  12: "&a&r "
  11: "&d%player%"
  10: " &7| &f\u0280\u1d00\u0274\u1d0b: &d%rank%"
  9: " &7| &f\u1d1b\u1d0f\u1d0b\u1d07\u0274\u0455: &d%tokens%"
  8: " &7| &f\u1d21\u026a\u0274\u0455: &d%wins%"
  7: " &7| &f\u0455\u1d1b\u1d00\u0280\u1d1b\u026a\u0274\u0262\u0020\u026a\u0274: &d%countdown%"
  6: "&l&r "
  5: "&d\u1d07\u1d20\u1d07\u0274\u1d1b\u0020"
  4: " &7| &f\u1d04\u1d1c\u0280\u0280\u1d07\u0274\u1d1b \u1d07\u1d20\u1d07\u0274\u1d1b: &d%event%"
  3: " &7| &f\u1d00\u029f\u026a\u1d20\u1d07: &d%alive%"
  2: " &7| &f\u1d05\u1d07\u1d00\u1d05: &d%dead%"
  1: "&r "
  0: "&d%CORE_ICON% &d\u1d04\u029c\u1d07\u0280\u0280\u028f\u0020&f\u0299\u029f\u1d0f\u0455\u0455\u1d0f\u1d0d &d[\u1d1b\u029c\u1d07\u1d0d\u1d07]" # bottom line