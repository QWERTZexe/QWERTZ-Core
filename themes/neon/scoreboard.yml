## QWERTZ Core Scoreboard Configuration
## https://modrinth.com/plugin/qwertz-core

## 0 is the bottom line, higher numbers are higher on the scoreboard.
## Set a line to "" to effectively remove it.  Use color codes (&)

# Valid placeholders: %server%, %event%, %alive%, %dead%, %countdown%, %player%, %rank%, %wins%, %tokens%
# Also all placeholders in "messages.yml"

# TO MAKE USE OF THIS FILE YOU MUST SET active-theme: "file" IN messages.yml!

title: "&5%server%" # Scoreboard Title

lines:
  12: "&a&r "
  11: "&5%player%"
  10: " &8| &7\u1d1b\u1d0f\u1d0b\u1d07\u0274\ua731: &5%tokens%"
  9: " &8| &7\ua731\u1d1b\u1d00\u0280\u1d1b\u026a\u0274\u0262\u0020\u026a\u0274: &5%countdown%"
  6: "&l&r "
  5: "&5êœ±\ua731\u1d07\u0280\u1d20\u1d07\u0280"
  4: " &8| &7\u1d07\u1d20\u1d07\u0274\u1d1b: &5%event%"
  3: " &8| &7\u1d00\u029f\u026a\u1d20\u1d07: &5%alive%"
  2: " &8| &7\u1d05\u1d07\u1d00\u1d05: &5%dead%"
  1: "&r "
  0: "&5%CORE_ICON% Neon [theme]" # Bottom line
