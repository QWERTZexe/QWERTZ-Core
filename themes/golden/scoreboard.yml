## QWERTZ Core Scoreboard Configuration
## https://modrinth.com/plugin/qwertz-core

## 0 is the bottom line, higher numbers are higher on the scoreboard.
## Set a line to "" to effectively remove it.  Use color codes (&)

# Valid placeholders: %server%, %event%, %alive%, %dead%, %countdown%, %player%, %rank%, %wins%, %tokens%
# Also all placeholders in "messages.yml"

# TO MAKE USE OF THIS FILE YOU MUST SET active-theme: "file" IN messages.yml!

title: "&e\ud83d\udde1&e&l %server% &e\ud83d\udde1" # Scoreboard Title

# The number color to use for the scoreboard numbers. To remove numbers set to "#"
# Hex colors only! e.g. "#ff00ff"
number-color: "#"

lines:
  12: "&a"  # Strikethrough line
  11: "&6Event info:"
  10: "&7| &eName: &6%event%"
  9:  "&7| &eAlive: &6%alive%"
  8:  ""
  7:  "&7| &eStarting in: &6%countdown%"
  6:  " " # Empty line
  5:  "&6%player%"
  4:  "&7| &eRank: &6%rank%"
  3:  "&7| &eWins: &6%wins%"
  2:  "&7| &eRevive Tokens: &6%tokens%"
  1:  "&6" # Strikethrough line
  0:   "" # Bottom line

